js/components/LoginScreen.js

```diff
@@ -148,8 +150,8 @@ class LoginScreen extends React.Component {
           ]}
         />
         <View style={{ height: 100, paddingTop: 20 }}>
-          <H2 style={titleTextStyles}>benvenuto nella tua</H2>
-          <H1 style={titleTextStyles}>Cittadinanza Digitale</H1>
+          <H2 style={titleTextStyles}>{strings('login.welcome2')}</H2>
```

Comment:
the order of these identifiers may be confusing, perhaps `login.welcome.line1` and `login.welcome.line2` would be less confusing